src/Fixtures/EmptySearchableModel.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Sti3bas\ScoutArray\Tests\Fixtures;
```

Comment:
Hmm... I'm not happy with this. Would like to keep it inside `tests` directory as these classes are only used for tests. Maybe there is some kind of workaround?

Reply:
I managed to resolve this issue, so the fixtures are now correctly back within `Test\Fixtures`